### ISSUE TITLE
Also escape `)` in `code` URL parameter

### ISSUE THREAD

**Issue by nomeata** [leanprover-community/lean4web#30]
I just had issues embedding a https://live.lean-lang.org/ url with a `code` parameter in markdown, presumably because the code contained a closing parenthesis, and thus the URL, and this confused the markdown parser.

For example
```
https://live.lean-lang.org/#project=lean-nightly&code=%23check%20()
```
I wonder if this problem would go away if lean4web would also URL-encode the parentheses, e.g.
```
https://live.lean-lang.org/#project=lean-nightly&code=%23check%20%28%29
```
These URLs already work, they just have to be generated as such.

**Comment by joneugster** [comment]
I looked at this once before, for a similar issue. 

In particular, the relevant code snippet is to replace `encodeURIComponent(content)` in this [line](https://github.com/leanprover-community/lean4web/blob/3cd85038d08ef6d70f2e2cd2656e967baead930b/client/src/App.tsx#L134) with a custom function that would also escape `%28` and `%29`.

I think I had a problem that even with this replacement the browser still changed it back, and I'm not sure if this is a browser feature, or if I did something wrong. Maybe it's worth trying again!
